src/wrover_status_light/ServiceIntro.md

```diff
@@ -0,0 +1,160 @@
+# About ROS Services
+## What is a ROS Service?
+Services are another form of communication between different software componenents, or more fomally known as nodes, that ROS offers. Common uses of services include sending requests for specific pieces of data, or notify another node to perform specific actions. 
+
+## What is the Makeup of a ROS Service?
+
+ROS services follow a client/server structure, where one node acts as a client that sends a request and another node acts as a server that receives the request and provides a response.
+
+Here is a list of key components involved in services:
+* Service definition: services are defined using a srv file. Similar to msg files for ROS topics, the srv file defines the name of a service, the data structure of the request message, and the data structure of the response message. 
+* Client node: the client node in responsible for initiating the service request. It creates the request message, populates it with the required data, and sends it to the server node. the client then waits for the server to process the request and replies with a response.
```

Comment:
```suggestion
* Client node: the client node in responsible for initiating the service request. It creates the request message, populates it with the required data, and sends it to the server node. The client then waits for the server to process the request and replies with a response.
```

Reply:
Addressed in commit [f4eb2b6](https://github.com/WisconsinRobotics/WRoverPlayground/pull/9/commits/f4eb2b6978e8c4699a10c0750eac8bd1e979b5e4).